bin/setup

```diff
@@ -117,8 +113,16 @@ has_node_version (){
 }
 
 install_nodejs (){
+  is_darwin || {
+    echo "Please install node for your platform."
+    exit 1
+  }
+
```

Comment:
Added this bit, so that it does not attempt to `brew install nvm` on Linux machines.